content/07.supplemental.md

```diff
@@ -8,6 +8,6 @@
 
 ![Figure S3. Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.](images/supplement/Figure-S3.png){#fig:FigS3}
 
-![Figure S4. Related to Figure 4. First two dimensions from UMAP of sample transcriptome data with points  colored by `molecular_subtype` for A, medulloblastoma, B, ependymoma, C, low-grade glioma, and D, high-grade diffuse astrocytic tumors.](images/supplement/Figure-S4.png){#fig:FigS4}
+![**Figure S4. Related to Figure 3.** A, Sample-specific RefSig signature weights across cancer groups ordered by decreasing Signature 1 exposure. B, Proportion of Signature 1 plotted by phase of therapy for each cancer group.](images/supplement/Figure-S4.png){#fig:FigS4}
 
-![Figure S5. Related to Figure 5.](images/supplement/Figure-S5.png){#fig:FigS5}
+![Figure S5. Related to Figures 4 & 5. First two dimensions from UMAP of sample transcriptome data with points  colored by `molecular_subtype` for A, medulloblastoma, B, ependymoma, C, low-grade glioma, and D, high-grade diffuse astrocytic tumors.](images/supplement/Figure-S5.png){#fig:FigS5}
```

Comment:
I think this should be Figure S4, not S5?

Reply:
I had so many changes going on that I didn't know where to put it. It used to be S4, but now S5. Let me drop that new S5 in here now since I change the text referring to it.

Reply:
> oncoprints

done

Reply:
updated